src/app/meshes/views/MeshView.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <KTabs
+    :tabs="items"
+    :model-value="(items.find(item => (router.currentRoute?.value.name ?? '').toString().startsWith(item.hash)) ?? items[0]).hash"
+    @click="change"
+  >
+    <template #mesh-detail-view-anchor>
+      <router-link
+        :to="{
+          name: items[0].hash,
+        }"
+      >
+        {{ items[0].title }}
+      </router-link>
+    </template>
+    <template #mesh-detail-view>
+      <RouterView
+        v-slot="routerView"
+      >
+        <component
+          :is="routerView.Component"
+          :key="routerView.route.path"
+        />
+      </RouterView>
+    </template>
+    <template #services-list-view-anchor>
+      <router-link
+        :to="{
+          name: items[1].hash,
+        }"
+      >
+        {{ items[1].title }}
+      </router-link>
+    </template>
+    <template #services-list-view>
+      <RouterView
+        v-slot="routerView"
+      >
+        <component
+          :is="routerView.Component"
+          :key="routerView.route.path"
+        />
+      </RouterView>
+    </template>
+    <template #gateways-list-view-anchor>
+      <router-link
+        :to="{
+          name: items[2].hash,
+        }"
+      >
+        {{ items[2].title }}
+      </router-link>
+    </template>
+    <template #gateways-list-view>
+      <RouterView
+        v-slot="routerView"
+      >
+        <component
+          :is="routerView.Component"
+          :key="routerView.route.path"
+        />
+      </RouterView>
+    </template>
+    <template #data-planes-list-view-anchor>
+      <router-link
+        :to="{
+          name: items[3].hash,
+        }"
+      >
+        {{ items[3].title }}
+      </router-link>
+    </template>
+    <template #data-planes-list-view>
+      <RouterView
+        v-slot="routerView"
+      >
+        <component
+          :is="routerView.Component"
+          :key="routerView.route.path"
+        />
+      </RouterView>
+    </template>
+    <template #policies-anchor>
+      <router-link
+        :to="{
+          name: items[4].hash,
+        }"
+      >
+        {{ items[4].title }}
+      </router-link>
+    </template>
+    <template #policies>
+      <RouterView
+        v-slot="routerView"
+      >
+        <component
+          :is="routerView.Component"
+          :key="routerView.route.path"
+        />
+      </RouterView>
+    </template>
```

Comment:
**Suggestion**: You can make this dynamic like so:

```suggestion
    <template
      v-for="item in items"
      :key="`${item.hash}-anchor`"
      #[`${item.hash}-anchor`]
    >
      <router-link
        :to="{
          name: item.hash,
        }"
      >
        {{ item.title }}
      </router-link>
    </template>

    <template
      v-for="item in items"
      :key="item.hash"
      #[item.hash]
    >
      <RouterView v-slot="routerView">
        <component
          :is="routerView.Component"
          :key="routerView.route.path"
        />
      </RouterView>
    </template>
```

Reply:
Oh perfect! The `v-for` on the slot of course! ty!

Reply:
This has been changed up, thanks again!